web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -595,6 +595,9 @@ export type SecurityGroupRule = {
   toPort: string;
   // CIDRs contains a list of IP ranges that this rule applies to and a description for the value.
   cidrs: Cidr[];
+  // Groups is a list of rules that allow another security group referenced
+  // by ID.
```

Comment:
we are now preferring the asterisks kind of documentation, i think it's b/c it shows on hover variable name (and below too)
```suggestion
  /**
  * Groups is a list of rules that allow another security group referenced
  * by ID.
  */
```